configure.ac

```
@@ -106,7 +106,12 @@ AC_PATH_PROG(CCACHE,ccache)
 dnl This one is not currently used anywhere, thus not required:
 AC_PATH_PROG(XGETTEXT,xgettext)
 ZC_REQUIRE_PROG(HEXDUMP,hexdump)
-ZC_REQUIRE_TOOL(READELF,readelf)
+
+dnl readelf is Linux only
+if test x$TARGET_OS = xlinux; then
+  ZC_REQUIRE_TOOL(READELF,readelf)
+fi
+
```

I think the problem here is that this does not revert the original change, so now nothing is configured for non-Linux platforms. Let's try this instead:
```suggestion
AC_PATH_PROG(READELF,readelf)
```

---

Hm, my change compiles fine on macOS. I don't think the CI failures were related to this commit. But I can try your proposal when I'm back at my computer.

---

I tested your suggestion and it still doesn't affect macOS, so in that sense, bring it in!

---

Your suggestion works for macOS but I can't vouch for other platforms so I'll take your word for it :)